scripts/outer/kubectl-kubestellar-ensure-location

```diff
@@ -118,11 +118,6 @@ then kubectl ws "${kubectl_flags[@]}" "$imw"
      cwsi=$imw
 fi
 echo "current cwsi is $cwsi"
-if ! kubectl "${kubectl_flags[@]}" get apibinding "edge.kubestellar.io" &> /dev/null; then
-    kubectl kcp "${kubectl_flags[@]}" bind apiexport root:espw:edge.kubestellar.io
-    echo "bound apiexport root:espw:edge.kubestellar.io into $cwsi:edge.kubestellar.io"
-else echo "edge.kubestellar.io apibinding exists in workspace $cwsi"
-fi
```

Comment:
With these lines gone, what is getting `locations` and `synctargets` defined in ISes now?

Reply:
Yes, we need to ensure this. Let's use https://github.com/kubestellar/kubestellar/issues/1337#issuecomment-1832146503 to track it.

Reply:
My question is, why is this not a fatal problem right now? For example, something that prevents this PR from passing CI?

Reply:
I found it. `kubestellar init` defines locations and synctargets in the ISes (IMWs) that _it_ creates. The existing tests only use the IS created by `kubestellar init`. So this PR is good enough to pass CI tests but has a bug that needs to be fixed.